File: Brute Force/bubblesort.py
# Selection Sort
# code by Akshat Aryan
import numpy as np

def bubblesort(arr):
    for i in range(0, len(arr) - 1):
        for j in range(0, len(arr) - i - 1):
            if(arr[j + 1] < arr[j]):
                (arr[j], arr[j + 1]) = (arr[j + 1], arr[j])

if(__name__ == "__main__"):
    arr = np.array(list(map(int, input("Enter the array:").split())))
    bubblesort(arr)
    print("Sorted array:\n", arr)

File: Brute Force/bubblesort.c
// Selection Sort
// code by Akshat Aryan
#include <stdio.h>

void swap(int *a, int *b){
    int temp = *a;
    *a = *b;
    *b = temp;
}

void bubblesort(int arr[], int n){
    for(int i = 0; i < n - 1; i++){
        for(int j = 0; j < n - i - 1; j++){
            if(arr[j + 1] < arr[j]){
                swap(&arr[j], &arr[j+1]);
            }
        }
    }
}

int main(){
    int n;
    printf("Enter the number of elements in the array:");
    scanf("%d", &n);
    int arr[n];
    printf("Enter the array:");
    for(int i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }
    bubblesort(arr, n);
    printf("Sorted array is:\n");
    for(int i = 0; i < n; i++){
        printf("%d ", arr[i]);
    }
    return 0;
}
